data/brands/amenity/pharmacy.json

```diff
@@ -1138,9 +1124,9 @@
       "displayName": "Farmacias Similares",
       "id": "farmaciassimilares-f147d5",
       "locationSet": {
-        "include": ["cl", "gt", "mx"]
+        "include": ["mx"]
       },
-      "matchNames": ["similares"],
+      "matchNames": ["Farmacia Similares"],
```

Comment:
If you run `npm run build` in the repo it will fix the case here to be lowercase

Reply:
Fixed. I did it manually because I'm in a pc without npm installed.